tests/test_annotations.py

```diff
@@ -37,12 +40,22 @@ def test_tags():
     bad_confidence = np.array([1, 0.5, -0.2])
     pytest.raises(ValueError, annotations.Tags, labels, bad_confidence)
 
+    # test units
+    bad_binary_confidence = np.array([1.0, 0.5, 0.0])
+    pytest.raises(ValueError, annotations.Tags, labels, bad_binary_confidence)
+
+    with pytest.raises(ValueError):
+        annotations.Tags(labels, confidence, labels_unit="bad_unit")
+
+    with pytest.raises(ValueError):
+        annotations.Tags(labels, confidence, confidence_unit="bad_unit")
+
 
 def test_events():
     # test good data
     intervals = np.array([[1.0, 2.0], [1.5, 3.0], [2.0, 3.0]])
     labels = ["Siren", "Laughter", "Engine"]
-    confidence = np.array([1, 0.5, 0.2])
+    confidence = np.array([1.0, 0.0, 1.0])
```

Comment:
just checking - do we have other tests that use confidence values other than 1/0 after this change?